src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -315,6 +315,7 @@ public function createAction(Request $request)
      * Show Employee edit page.
      *
      * @DemoRestricted(redirectRoute="admin_employees_index")
+     * @AdminSecurity("is_granted('update', request.get('_legacy_controller'))")
```

Comment:
Looks like there is a custom rule 🤔 
```
        $formAccessChecker = $this->get('prestashop.adapter.employee.form_access_checker');

        if (!$formAccessChecker->canAccessEditFormFor($employeeId)) {
``` 

So AdminSecurity here will probably not get the job done 😢 this is an exception to the general rule